expyfun/_eyelink_controller.py

```diff
@@ -381,6 +385,36 @@ def calibrate(self, beep=False, prompt=True):
         self._start_recording()
         return fname
 
+    def check_recalibrate(self, keys=None, prompt=True):
+        """Compare key buffer to recalibration keys and calibrate if matched.
+
+        This function always uses the keyboard, so is part of abstraction.
+
+        Parameters
+        ----------
+        keys : list or string
+            keys to check if prompt recalibration
+        prompt : bool
+            Whether to show the calibration prompt or not
```

Comment:
Whether to show the calibration prompt screen before starting the calibration procedure